### PR TITLE
Update parallel raytracing .gitignore

### PR DESCRIPTION
**Pull request by jthemphill**
Running the build script creates some `*.d.ts` files and a `snippets` folder. Adding these to the `.gitignore`.

### CODE REVIEW

File: examples/raytrace-parallel/.gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 raytrace_parallel.js
+raytrace_parallel.d.ts
 raytrace_parallel_bg.wasm
+raytrace_parallel_bg.wasm.d.ts
+snippets
```
Comment (daxpedda) [inline examples/raytrace-parallel/.gitignore:5]:
```suggestion
snippets

```

**Review by daxpedda — CHANGES_REQUESTED** [review]
Couldn't we use the `--no-typescript` flag instead?
`snippets` would still be required though.

**Comment by Liamolucko** [comment]
I'm not really sure why `raytrace-parallel` (and `wasm-audio-worklet`) puts its output into the current directory in the first place. All the other (non-Webpack-based) examples put their output into `pkg` instead: changing `raytrace-parallel` to do to the same would get rid of the need to even have this `.gitignore`, since `pkg` is already ignored by `examples/.gitignore`.

**Comment by jthemphill** [comment]
> All the other (non-Webpack-based) examples put their output into `pkg` instead: changing `raytrace-parallel` to do to the same would get rid of the need to even have this `.gitignore`, since `pkg` is already ignored by `examples/.gitignore`.

Sure! But since this now involves changing `build.sh` or the future `build.py`, I'm going to rebase this atop #3603 to avoid skew.

File: examples/raytrace-parallel/.gitignore [hunk 2]
```diff
@@ -1,2 +1 @@
-raytrace_parallel.js
-raytrace_parallel_bg.wasm
+snippets
```
Comment (daxpedda) [inline examples/raytrace-parallel/.gitignore:1]: I thought now that everything is in `pkg` we don't need the `.gitignore` at all?

File: examples/raytrace-parallel/index.html
Comment (daxpedda) [inline examples/raytrace-parallel/index.html:1]: Please leave out the HTML formatting from this PR. Same for the other file.

**Review by daxpedda — CHANGES_REQUESTED** [review]

File: examples/raytrace-parallel/.gitignore (hunk 2 above)
Comment (jthemphill) [reply examples/raytrace-parallel/.gitignore:1]: We still generate a `snippets/worker.js` file which is not in `pkg`. I'm not sure why it's there or how to move it into `pkg`. Do you not see this when you run `build.sh` or `build.py`?

Comment (daxpedda) [reply examples/raytrace-parallel/.gitignore:1]: Nope, for me the snippets folder lands in the `pkg` directory, as it should. Could you double check?

Comment (jthemphill) [reply examples/raytrace-parallel/.gitignore:1]: Oh, yes it is in `pkg` now!

**Review by daxpedda — APPROVED** [review]
Thank you!

**Comment by jthemphill** [comment]
> Sure! But since this now involves changing `build.sh` or the future `build.py`, I'm going to rebase this atop #3603 to avoid skew.

Oh, um... heads up that I had rebased this on #3603. This is because I thought #3603 would be merged first, and because the `.gitignore` depended on us storing everything in `pkg`, which happened in a change I made to `build.py`.

The end result of all this is that you have already merged most of #3603, probably without realizing it😬

**Comment by daxpedda** [comment]
... I sincerely apologize, I've somehow missed that, this is most definitely not your fault.
I will revert this so we can properly continue in #3603.

**Comment by daxpedda** [comment]
I included this change in #3603, so you don't have to re-open this.

**Pull request merged**